lib/Manager.php

```diff
@@ -1111,6 +1111,7 @@ public function createRoom(int $type, string $name = '', string $objectType = ''
 			'token' => $token,
 			'object_type' => $objectType,
 			'object_id' => $objectId,
+			'password' => $password
```

Comment:
this should be the hash aswell

Reply:
IMHO the responsibility of hashing should be with the insert.